LOGGING/test/EtwCollector.Tests/EtwCollector.Tests.csproj

```diff
@@ -2,9 +2,10 @@
 
   <PropertyGroup>
     <RootNamespace>Microsoft.ApplicationInsights.EtwCollector.Tests</RootNamespace>
-    <AssemblyName>Microsoft.ApplicationInsights.EtwCollector.Net46.Tests</AssemblyName>
-    <TargetFramework>net46</TargetFramework>
+    <AssemblyName>Microsoft.ApplicationInsights.EtwCollector.Tests</AssemblyName>
+    <TargetFrameworks>net46;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
```

Comment:
This project is only for .NET Framework so no need of testing for netcore targets.
https://github.com/microsoft/ApplicationInsights-dotnet/blob/develop/LOGGING/src/EtwCollector/EtwCollector.csproj#L3

Reply:
ok! thanks for reviewing